modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/editor/config/RichTextFragmentCKEditor5EditorConfigContributor.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.site.cms.site.initializer.internal.editor.config;
+
+import com.liferay.portal.kernel.editor.configuration.BaseEditorConfigContributor;
+import com.liferay.portal.kernel.editor.configuration.EditorConfigContributor;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.portlet.RequestBackedPortletURLFactory;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Marko Čikoš
+ */
+@Component(
+	property = "editor.config.key=richTextFragmentCKEditor5",
```

Comment:
This key scopes contributor to all Rich Text fragments. Origin of the key is [configuration in layout-taglib](https://github.com/liferay/liferay-portal/blob/master/modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/internal/editor/configuration/RichTextCKEditor5EditorConfigContributor.java#L23).

With just this, settings in this file would affect CMS site, but also any Rich Text fragment placed in page editor.

Reply:
This is a regular usage of the config contributor, so it's fine